jplag.frontend-utils/src/main/java/de/jplag/Token.java

```diff
@@ -79,27 +78,9 @@ public int getLine() {
     }
 
     /**
-     * @return the type identifier of the token.
+     * @return the type of the token.
      */
-    public int getType() {
+    public TokenType getType() {
         return type;
     }
-
-    /**
-     * Sets the character index which denotes where the code sections represented by this token starts in the line.
-     * @param column is the index in characters to set.
-     */
-    public void setColumn(int column) {
-        this.column = column;
-    }
-
-    @Override
-    public String toString() {
-        return type2string();
-    }
```

Comment:
This breaks the metamodel tree view in the EMF language module! The `toString()` method should be retained.

<details>

![grafik](https://user-images.githubusercontent.com/4396919/189845649-b29c0f6e-1b88-47dc-9dce-00f13fce1e2e.png)

</details>

Reply:
Resolved by adjusting the `MetamodelTreeView` to print the intended value instead of relying on the token serialization.